src/test/java/core/basesyntax/service/impl/ReportCreatorImplTest.java

```diff
@@ -10,7 +10,6 @@
 import org.junit.Test;
 
 public class ReportCreatorImplTest {
-    private static final String SEPARATOR = System.lineSeparator();
```

Comment:
sorry, my bad. if you are using line separator for test, so having it constant is better.